sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/AdtQueryBuilder.cs

```diff
@@ -89,27 +91,62 @@ public FromQuery SelectCount()
         /// </example>
         /// <param name="customQuery"> Query in string format. </param>
         /// <returns> Query that contains a select clause. </returns>
-        public FromQuery SelectCustom(string customQuery)
+        public SelectAsQuery SelectCustom(string customQuery)
         {
             return _selectQuery.SelectCustom(customQuery);
         }
 
+        /// <summary>
+        /// Used to alias selectable properties in place of the AS keyword.
+        /// </summary>
+        /// <param name="literal"> The proper name for the selectable property in the ADT Query Language. </param>
+        /// <param name="alias"> The alias to be assigned to the return contents in the query response. </param>
+        /// <returns> Query that contains an aliased select clause. </returns>
+        public SelectAsQuery SelectAs(string literal, string alias)
+        {
+            return _selectAs.SelectAs(literal, alias);
+        }
+
         /// <summary>
         /// Gets the string representation of the built query.
         /// </summary>
         /// <returns> String represenation of query. </returns>
         public string GetQueryText()
         {
             var finalQuery = new StringBuilder();
+            finalQuery.Append(QueryConstants.Select).Append(' ');
 
             // build the select string
-            finalQuery.Append(_selectQuery.GetQueryText());
+            string selectStatementQueryText = _selectQuery.GetQueryText();
+            string selectStatementAliasedQueryText = _selectAs.GetQueryText();
+
+            bool nonAliasedSelectStatements = selectStatementQueryText.Length > 0;
+            bool aliasedSelectStatements = selectStatementAliasedQueryText.Length > 0;
+
+            // begin with any non-aliased select components
+            finalQuery.Append(selectStatementQueryText);
```

Comment:
I think we should check if the `selectStatementQueryText` is null or empty and if it isn't , append it.